scrapy/linkextractors/__init__.py

```diff
@@ -28,7 +28,7 @@
 
     # video
     '3gp', 'asf', 'asx', 'avi', 'mov', 'mp4', 'mpg', 'qt', 'rm', 'swf', 'wmv',
-    'm4a', 'm4v',
+    'm4a', 'm4v', 'flv'
```

Comment:
Comma is missing. You've replaced "xls" with "flvxls" in this PR: Python joins strings with no separators between them.

Reply:
Ah, sorry. Added comma.